lektor/admin/static/js/widgets/ToastEditor.jsx

```diff
@@ -0,0 +1,213 @@
+'use strict'
```

Comment:
This file is call `ToastEditor` while everything else in this folder starts with a lower case name. The extensions are prefixed with `tui`. Shouldn't they have the same prefix?

Reply:
Maybe the editor and its extensions should be moved to a subdirectory of widgets? As the extensions aren't actually widgets themselves, so it doesn't make a whole lot of sense for them to be directly in widgets?

Reply:
I think a sub directory is good, that way to can skip the naming prefixes too.